README.md

```diff
@@ -139,7 +139,7 @@ A_rounded, b_rounded, Tr, Tr_shift = sampler.round_polytope(A, b, method="min_el
 Then, to sample from the rounded polytope, the user has to call the following static method of PolytopeSampler class,
 
 ```python
-samples = sample_from_polytope(A_rounded, b_rounded)
+sampler = sample_from_polytope(A_rounded, b_rounded)
```

Comment:
I think this is not correct. This is a set of samples (that also used below) not a sampler as the one returned for instance from `PolytopeSampler(model)`